packages/icons/icons.json

```diff
@@ -4373,5 +4373,110 @@
         "tags": "layer, map, reference, point, geography, distance, boundry",
         "group": "miscellaneous",
         "codepoint": 62321
+    },
+    {
+        "displayName": "Block promote",
+        "iconName": "block-promote",
+        "tags": "upgrade, line, chevrons",
+        "group": "interface",
+        "codepoint": 62322
+    },
+    {
+        "displayName": "Cubes",
+        "iconName": "cubes",
+        "tags": "blocks, shapes, hexagons, stack, geometry",
+        "group": "miscellaneous",
+        "codepoint": 62323
+    },
+    {
+        "displayName": "Hexagon",
+        "iconName": "hexagon",
+        "tags": "shape, geometry, polygon",
+        "group": "miscellaneous",
+        "codepoint": 62324
+    },
+    {
+        "displayName": "Package",
+        "iconName": "package",
+        "tags": "box, parcel, delivery, container, carton, archive, attachment",
+        "group": "file",
+        "codepoint": 62325
+    },
+    {
+        "displayName": "Pill",
+        "iconName": "pill",
+        "tags": "capsule, medicine, health, drug, pharmaceutical, label",
+        "group": "miscellaneous",
+        "codepoint": 62326
+    },
+    {
+        "displayName": "Server install",
+        "iconName": "server-install",
+        "tags": "setup, network, IT, data, hosting, hardware",
+        "group": "interface",
+        "codepoint": 62327
+    },
+    {
+        "displayName": "Server",
+        "iconName": "server",
+        "tags": "server, network, IT, data, hosting, hardware",
+        "group": "interface",
+        "codepoint": 62328
+    },
+    {
+        "displayName": "Tag add",
+        "iconName": "tag-add",
+        "tags": "label, badge, identification, plus, create",
+        "group": "action",
+        "codepoint": 62329
+    },
+    {
+        "displayName": "Tag promote",
+        "iconName": "tag-promote",
+        "tags": "label, badge, identification, upgrade",
```

Comment:
For any icon with the double chevrons, should we consider aligning with the same categories used for the `chevron-right` icon? Or some of them? (e.g. Arrow, direction, next, forward)